### ISSUE TITLE
Support spring 2.4.3

### ISSUE THREAD

**Issue by EugenMayer** [Blazebit/blaze-persistence#1266]
### Description
When using BP with Spring Boot to 2.4.3+ (works with 2.4.2) an exception is thrown.

### Expected behavior
No exception :)

### Actual behavior
```
  Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'requestMappingHandlerAdapter' defined in class path resource [org/springframework/boot/autoconfigure/web/servlet/WebMvcAutoConfiguration$EnableWebMvcConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter]: Factory method 'requestMappingHandlerAdapter' threw exception; nested exception is java.lang.NoSuchFieldError: com/blazebit/persistence/spring/data/webmvc/impl/json/EntityViewAwareMappingJackson2HttpMessageConverter.objectMapper

  Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter]: Factory method 'requestMappingHandlerAdapter' threw exception; nested exception is java.lang.NoSuchFieldError: com/blazebit/persistence/spring/data/webmvc/impl/json/EntityViewAwareMappingJackson2HttpMessageConverter.objectMapper

  Caused by: java.lang.NoSuchFieldError: com/blazebit/persistence/spring/data/webmvc/impl/json/EntityViewAwareMappingJackson2HttpMessageConverter.objectMapper
```
### Environment

Version:  1.6.0-Alpha2	           
JPA-Provider: Hibernate 
DBMS:  Mariadb 10.3

**Comment by EugenMayer** [comment]
Fairly simple to fix i guess - you are using the inherited field directly, but they renamed it to `defaultObjectMapper`. You should just migrate to `getObjectMapper()` and be compatible with pre <=2.4.2 and higher

**Comment by beikov** [comment]
Merged via 0cbe70e3b8070c7879de64d472e29340ec81106b
